.codespell_words

```diff
@@ -0,0 +1,18 @@
+ang
+ans
+atleast
+ba
+dof
+dur
+iff
+keyserver
+nto
+ot
+parameterizes
+planed
+tork
+uint
+whis
```

Comment:
NIT: do we need these? I think they were meant to be used with the library since we have them as variable names there